summershum/model.py

```diff
@@ -7,6 +7,39 @@
 
 BASE = declarative_base()
 
+def create_tables(db_url, debug=False):
+    """ Create the tables in the database using the information from the
+    url obtained.
+
+    :arg db_url, URL used to connect to the database. The URL contains
+        information with regards to the database engine, the host to
+        connect to, the user and password and the database name.
+          ie: <engine>://<user>:<password>@<host>/<dbname>
+    :kwarg alembic_ini, path to the alembic ini file. This is necessary
+        to be able to use alembic correctly, but not for the unit-tests.
+    :kwarg debug, a boolean specifying wether we should have the verbose
+        output of sqlalchemy or not.
+    :return a session that can be used to query the database.
+
+    """
+    engine = sa.create_engine(db_url, echo=debug)
+    models.Base.metadata.create_all(engine)
+    # engine.execute(collection_package_create_view(driver=engine.driver))
+    if db_url.startswith('sqlite:'):
+        # Ignore the warning about con_record
+        # pylint: disable=W0613
+        def _fk_pragma_on_connect(dbapi_con, con_record):
+            ''' Tries to enforce referential constraints on sqlite. '''
+            dbapi_con.execute('pragma foreign_keys=ON')
+        sa.event.listen(engine, 'connect', _fk_pragma_on_connect)
+
+    # Generate the Alembic version table and "stamp" it with the latest rev
+    alembic.command.stamp(get_alembic_config(db_url), "head")
```

Comment:
@pypingou  This is from https://github.com/fedora-infra/pkgdb2/blob/master/pkgdb2/lib/model/__init__.py#L69. This was just for experimentation, later on I will make changes in it.